bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -189,6 +211,115 @@ const VideoListItem = (props) => {
       </Styled.BottomBar>
     </>
   );
+  const resetEvent = (e) => {
+    e.preventDefault();
+    e.stopPropagation();
+  }
+
+  useEffect(() => {
+    const onDragOver = (e) => {
+      resetEvent(e);
+      setDragging(true);
+    };
+    const onDragLeave = (e) => {
+      resetEvent(e);
+      setDragging(false);
+    };
+    const onDrop = (e) => {
+      resetEvent(e);
+      setDragging(false);
+    };
+
+    window.addEventListener('dragover', onDragOver);
+    window.addEventListener('dragleave', onDragLeave);
+    window.addEventListener('drop', onDrop);
+
+    return () => {
+      window.removeEventListener('dragover', onDragOver);
+      window.removeEventListener('dragleave', onDragLeave);
+      window.removeEventListener('drop', onDrop);
+    };
+  }, []);
+
+  const startAndSaveVirtualBackground = (file) => {
+    const { name: filename } = file;
+    const reader = new FileReader();
```

Comment:
This is method is very similar from the one in video-preview/virtual-background/component counterpart. There's some room for centralization here, specially the file name parsing thing. So maybe centralize it in the new vbg service file?